src/networkcloud/azext_networkcloud/operations/common_ssh.py

```diff
@@ -0,0 +1,144 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Custom code that is added in addition to auto-generated by aaz-dev code.
+# --------------------------------------------------------------------------------------------
+"""
+Provides Ssh Key customization
+"""
+# pylint: disable=line-too-long,wildcard-import,too-many-nested-blocks,no-self-argument,no-method-argument
+
+import os
+
+from azure.cli.core import keys
+from azure.cli.core.aaz import AAZBoolArg, AAZListArg, AAZStrArg
+from azure.cli.core.azclierror import InvalidArgumentValueError
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+class CustomSshOptions:
+    """Custom class providing common ssh operations"""
+
+    @staticmethod
+    def generate_ssh_keys():
+        """generates ssh keys in user .ssh folder"""
+
+        key_name = "id_rsa"
```

Comment:
I think the `id_rsa` name is commonly used. Will the generation overwrite the existing keys?

Reply:
Updated.